### PR TITLE
Reformatting constant_pad

### LINKED ISSUE #6448: constant_pad

### PR DESCRIPTION
**Pull request by volkaseezn**
Close #6448

### CODE REVIEW

**Comment by ahmedo42** [comment]
Hi @volkaseezn, Thanks for your contribution!
you should start by fixing the lint errors by using `pre-commit` as mentioned in the contributing guide and then go through the checklist of items to complete for `constant_pad`, for example, you should write examples for the container and instance methods in their respective files. 

you should request a review from me once you're done and I'll get to it ASAP

**Comment by volkaseezn** [comment]
Thanks @ahmedo42. May I get some assistance with the checklist of items to complete? I'm having difficulty finding that

File: ivy/array/manipulation.py [hunk 1]
```diff
@@ -351,6 +351,13 @@ def constant_pad(
         ivy.Array instance method variant of ivy.constant_pad. This method simply
         wraps the function, and so the docstring for ivy.constant_pad also applies
         to this method with minimal changes.
+
+        Examples
+        --------
+        >>> x = ivy.array([-4.7, -2.3, 0.7]) #x.shape->(3,)
+        >>> y = x.expand_dims() #y.shape->(1, 3)
```
Comment (ahmedo42) [inline ivy/array/manipulation.py:358]: it seems like you copied over an example from a different function

File: ivy/container/manipulation.py [hunk 2]
```diff
@@ -1295,6 +1295,21 @@ def constant_pad(
...
         wraps the function, and so the docstring for ivy.constant_pad also applies to
         this method with minimal changes.
+
+        Examples
+        --------
+        >>> x = ivy.Container(a=ivy.array([[0., 1.],
+                               [2., 3.]]),
+                  b=ivy.array([[4., 5.],
+                               [6., 7.]]))
+        >>> y = x.expand_dims(axis=1)
```
Comment (ahmedo42) [inline ivy/container/manipulation.py:1305]: same here , you should also add examples for the container static method

**Review by ahmedo42 — CHANGES_REQUESTED** [review]

File: ivy/array/manipulation.py (hunk 1 above)
Comment (volkaseezn) [reply ivy/array/manipulation.py:358]: Huge mistakes. I'd be on it shortly

**Comment by ahmedo42** [comment]
The docstring examples you provided are failing the docstring tests, you should fix those.
you can check by running the docstring tests locally, checking the logs of the CI, or simply running the example in the terminal

**Comment by volkaseezn** [comment]
Hi @ahmedo42 , I've been looking at the logs and I don't seem to be able to see the problem. Could you assist me with it?

**Comment by ahmedo42** [comment]
@volkaseezn kindly refer to the "Docstring tests" section in [this page](https://lets-unify.ai/ivy/deep_dive/docstring_examples.html)

**Review by ahmedo42 — CHANGES_REQUESTED** [review]
Looking good @volkaseezn, two examples should be added, and then finally we can merge 😄 
1. using the out argument
2. using the value argument

**Comment by volkaseezn** [comment]
On it!

**Comment by volkaseezn** [comment]
Hey @ahmedo42 please have a look

**Comment by ahmedo42** [comment]
Thanks, @volkaseezn 
LGTM!

**Pull request merged**